test/units/modules/storage/netapp/test_na_ontap_gather_facts.py

```diff
@@ -123,8 +123,15 @@ def get_gather_facts_mock_object(self, kind=None):
         """
         Helper method to return an na_ontap_gather_facts object
         """
+
+        argument_spec = netapp_utils.na_ontap_host_argument_spec()
+        argument_spec.update(dict(
+            state=dict(default='info', choices=['info']),
+            gather_subset=dict(default=['all'], type='list'),
+            max_records=dict(default='1024', type='int'),
```

Comment:
Quotes?

Reply:
... and here

Reply:
Since its an int i think default has to be 1024 with out quotes.

Reply:
yes, sorry, i forgot to add testing module to this commit.